libraries/chain/include/eosio/chain/host_context.hpp

```diff
@@ -582,6 +583,7 @@ class host_context {
    controller&                      control;
    chainbase::database&             db;  ///< database where state is stored
    transaction_context&             trx_context; ///< transaction context in which the action is running
+   uint32_t                         recurse_depth; ///< how deep inline actions can recurse
```

Comment:
I thought sync calls can't make inline action calls. Why is this here?

Reply:
https://github.com/AntelopeIO/spring/pull/1285/commits/7d16b88cf172aa95a78344e0d3c17f9aea463426